src/components/SoloArticle.jsx

```diff
@@ -1,22 +1,39 @@
 import { useEffect, useState } from "react";
 import { getArticleById } from "../utils/getArticleById";
+import { patchVotes } from "../utils/patchVotes";
+
 
 const SoloArticle = ({ article_id }) => {
 
     const [singleArticle, setSingleArticle] = useState([]);
     const [isLoading, setIsLoading] = useState(true)
+    const [voteCount, setVoteCount] = useState()
+    const [err, setErr] = useState(null)
 
     useEffect(() => {
         getArticleById(article_id).then(({ article }) => {
             setSingleArticle(article)
+            setVoteCount(article.votes)
             setIsLoading(false)
         }).catch((err) => {
-            console.log(err, "DEAL WITH ERROR HERE");
+
         })
 
     }, [])
 
 
+    const handleClick = (event) => {
+        event.preventDefault()
+        setVoteCount(voteCount + 1)
+        setErr(null)
+        patchVotes(article_id)
+            .catch((err) => {
```

Comment:
I dont think you can ever get into this catch block because in your patch article function you can never throw an error because you handle it in there and return it as a value